src/herder/HerderImpl.cpp

```diff
@@ -399,13 +404,47 @@ HerderImpl::writeDebugTxSet(LedgerCloseData const& lcd)
     }
 }
 
+void
+recordExternalizeAndCheckCloseTimeDrift(
+    uint64 slotIndex, StellarValue const& value,
+    std::map<uint32_t, std::pair<uint64_t, std::optional<uint64_t>>>& ctMap)
+{
+    auto it = ctMap.find(slotIndex);
+    if (it != ctMap.end())
+    {
+        it->second.second = value.closeTime;
+    }
+
+    if (ctMap.size() >= CLOSE_TIME_DRIFT_LEDGER_WINDOW)
+    {
+        medida::Histogram h(medida::SamplingInterface::SampleType::kSliding);
```

Comment:
A comment here for this histogram would be helpful, it's pretty unclear what's happening unless you look up the definition of ctMap because of the `ct.second.second` lines.

Reply:
I re-wrote the loop with the use of structured bindings, I think that should make it more clear than a comment. Lmk what you think though.

Reply:
Great, thanks!